media/pages/sk/podporte-nas/page.html

```diff
@@ -28,60 +28,61 @@ <h3>2% z daní</h3>
-        2020 pošlite/doručte formuláre daňovému úradu v mieste vášho bydliska.
+        Ak ste <strong>zamestnanec</strong>, požiadajte do 15.2.2020 zamestnávateľa o vykonanie
+        ročného zúčtovania zaplatených preddavkov na daň. Potom požiadajte zamestnávateľa
+        o vystavenie tlačiva <em>Potvrdenie o zaplatení dane</em>. Z tohto potvrdenia si viete
+        zistiť dátum zaplatenia dane a vypočítať 2% z Vašej zaplatenej dane
+        (to musí byť minimálne 3 €). Potrebné údaje vpíšte do tlačiva
+        <a href="{% page 'zamestnanci-chcemvediet.pdf' %}">Vyhlásenie o poukázaní 2%</a>. Najneskôr
+        do 30. apríla 2020 pošlite/doručte formuláre daňovému úradu v mieste vášho bydliska.
       </li>
     </ul>
   </div>
-</div>
-
-<h2>Aké sú naše výsledky?</h2>
-
-<div class="row">
-  <div class="col-sm-6">
+  <div class="col-md-6">
+    <h2>Aké sú naše výsledky?</h2>
     <p>
-      Od roku 2016 je portál Chcemvediet.sk prostriedkom k tomu, aby bežní ľudia mohli pohodlne podávať infožiadosti.
-      Medzi hlavné metriky, ktoré sledujeme, patrí počet <strong>registrovaných užívateľov</strong>, <strong>podaných
-      infožiadostí</strong> a počet <strong>inštitúcií v databáze</strong>.
+      Od roku 2016 je portál Chcemvediet.sk prostriedkom k tomu, aby bežní ľudia mohli pohodlne
+      podávať infožiadosti. Medzi hlavné metriky, ktoré sledujeme, patrí počet
+      <strong>registrovaných užívateľov</strong>, <strong>podaných infožiadostí</strong> a počet
+      <strong>inštitúcií v databáze</strong>.
     </p>
     <p>
       V roku 2017 sme sa spojili s ďalšími iniciatívnymi programátormi a tak vznikla platforma
-      <a href="https://verejne.digital">verejne.digital</a>. Na nej sme začali analyzovať prepojenia politikov a firiem
-      navzájom, a zobrazovať tieto prepojenia ako pekné grafy. Tiež sme analyzovali verejné obstarávania, kde hľadáme
-      podobné obstarávania, firmy, ktoré sa ich zúčastnili, a kontrolujeme, či sa cena obstarávania nevymyká doterajším
-      podobným obstarávaniam. Neskôr pribudol majetkový profil politikov na základe ich majetkových priznaní, spolu
-      s porovnaním voči údajom z katastra. Pred prezidentskými voľbami 209 sme doplnili majetkový profil kandidátov na
-      prezidenta.
+      <a href="https://verejne.digital">verejne.digital</a>. Na nej sme začali analyzovať prepojenia
+      politikov a firiem navzájom, a zobrazovať tieto prepojenia ako pekné grafy. Tiež sme
+      analyzovali verejné obstarávania, kde hľadáme podobné obstarávania, firmy, ktoré sa ich
+      zúčastnili, a kontrolujeme, či sa cena obstarávania nevymyká doterajším podobným
+      obstarávaniam. Neskôr pribudol majetkový profil politikov na základe ich majetkových priznaní,
+      spolu s porovnaním voči údajom z katastra. Pred prezidentskými voľbami 2019 sme doplnili
+      majetkový profil kandidátov na prezidenta.
     </p>
     Výročné správy:
```

Comment:
Na riadku "Výročné správy:" nesedi baseline:

![vyrocne-spravy](https://user-images.githubusercontent.com/5547275/119360211-ac882d80-bcaa-11eb-82db-b2478e4d8e58.png)

Bude to tym, ze nie je to odstavec/nadpis, ale iba riadok nadivoko bez typografickeho obalu.

Sprav z toho asi `<h3>`.

Reply:
:ok: V nadpise sa mi nepaci dvojbodka, dal som ju prec.

Reply:
Ano, dvojbodka v nadpise nema co robit. Vdaka